examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -370,6 +387,12 @@ def parse_args(input_args=None):
             " `args.validation_prompt` multiple times: `args.num_validation_images`."
         ),
     )
+    parser.add_argument(
+        "--do_edm_style_training",
+        default=False,
```

Comment:
No need.

Reply:
it's not? 🤔😅

Reply:
I meant no need for any default. If you don't specify that flag while launching your command, it will be false. That's how `store_true` works.